commands/cat_at.py

```diff
@@ -54,4 +51,51 @@ def run(self):
             end_order = last_append_sector_order
             for sector in end_sectors:
                 end_order += 1
-                sector.order = end_order
+                SectorDao.insert_sector_data(
+                    sector, data=sector.data,
+                    order=end_order, file_id=sector.file_id)
+
+    def read_from_file(self, file, index, size):
```

Comment:
This would be better in FileDao, possibly a single function where to read entire file, index = 0.